packages/design-system-docs/src/pages/startup/child-design-systems.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Child Design Systems
+weight: 6
+---
+
+The CMS design system (CMSDS) provides a set of components and styles to meet many design and development needs. However, there are cases where product teams need to modify a design system beyond [theming]({{root}}/startup/theming/) and publish a child design system for other product teams to consume.
+
+## What is a child design systems?
+
+A child design system is a design system built off of the core CMSDS for a specific context. It contains custom assets (components, styles, etc..) and is published via NPM to be consumed by multiple product teams. A child design system can contain new components, modifications to existing core components, or modifications to styles. Child design systems import the core CMSDS as a dependency which allows for receiving updates, using developer tools, scripts, and documentation from the core CMSDS to manage their own design system.
```

Comment:
It could be helpful to call out that these were previously referred to as site packages for folks who aren't aware the nomenclature has changed.

Reply:
This could also be a good place to link to documentation on guidance for adding new components & modifications to the core CMSDS.